pom.xml

```diff
@@ -7,7 +7,7 @@
   <groupId>ir.sahab</groupId>
   <artifactId>spring-boot-rest-commons</artifactId>
   <name>Spring Boot Rest Commons</name>
-  <version>1.0.1</version>
+  <version>1.0.2</version>
```

Comment:
1.1.0

According to changes I have suggested, it can be considered as a feature: default value for base path